jf_agent/main.py

```diff
@@ -492,6 +494,11 @@ def upload_file(filename, path_to_obj, signed_url):
         )
         return
 
+    # If sending agent config flag is on, upload config.yml to s3 bucket
```

Comment:
There's reason why this can't be done as another file to be uploaded by `upload_file_from_thread` in the block around line 476, is there?

Reply:
Now that you mention it, I probably could add config.yml to the `filenames` list before that line. This just seemed more explicit to me.